dependencies/jessie/foreman-tasks-core/changelog

```diff
@@ -0,0 +1,5 @@
+ruby-foreman-tasks-core (0.1.1-1) stable; urgency=low
```

Comment:
What name do you think would better fit here? I'm open to suggestions, although I would prefer not to change that right away.